sdk/nodejs/runtime/rpc.ts

```diff
@@ -589,6 +589,18 @@ export function unwrapRpcSecret(obj: any): any {
     return obj.value;
 }
 
+export function containsUnknownValues(value: any): boolean {
```

Comment:
```suggestion
/** @internal */
export function containsUnknownValues(value: any): boolean {
```

Reply:
Feels like we could type `value: unknown` without any code changes, and that'd be much better than `any` -- thoughts?

Reply:
No because this is recursive, so you can pass a non unknown in as the param and get true back

Reply:
Ignore me I was thinking our unknown type, not the TS annotation. Will is right this should take unknown not any.